.cbmc-batch/include/cbmc_invariants.h

```diff
@@ -23,10 +23,10 @@
  * API. However, we can check them in the proof harnesses because these properties are modeled in our abstract model of
  * OpenSSL. */
 
-/* Checks that the message-digest context is valid based on the state of its members. */
+/** Checks that the message-digest context is valid based on the state of its members. */
 bool aws_cryptosdk_md_context_is_valid_cbmc(struct aws_cryptosdk_md_context *md_context);
```

Comment:
Not going to block merging this PR for this, but instead of having a separate function, it would make sense to use an ifdef and only have one function name.  Then we can still include it in the functional preconditions.